build.gradle

```diff
@@ -73,4 +76,33 @@ sonar {
 
 tasks.named('sonar').configure {
 	dependsOn jacocoTestReport
+}
+
+spotless {
+	java {
```

Comment:
we can simplify the java conf. using the google format to java:
https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#google-java-format

p.s: using only default, the optional conf. is not necessary